calabash-cucumber/lib/calabash-cucumber/usage_tracker.rb

```diff
@@ -56,6 +57,21 @@ def post_usage_async
       private
 
       # @!visibility private
+      def preferences
+        @preferences ||= Calabash::Cucumber::Preferences.new
+      end
+
+      # @!visibility private
+      def user_id
+        @user_id ||= preferences.user_id
+      end
+
+      # @!visibility private
+      def allowed_to_track
+        @allowed_to_track ||= preferences.usage_tracking
+      end
+
+      # @!visibility private
       def self.track_usage?
         @@track_usage && !self.xtc?
```

Comment:
What's the difference between `@@track_usage` and `@allowed_to_track` ? Could we not have one variable serve both purposes.
e.g,
```ruby
def self.track_usage?
    @@track_usage ||= preferences.usage_tracking != "none" && !self.xtc?
```